README.md

```diff
@@ -37,6 +37,30 @@ signatures:
 
 This policy will validate all images with the public keys provided, and images whose name matches `ghcr.io/kubewarden/*` with the keyless provided.
 
+Another example that validates a GitHub action:
+
+```
+signatures:
+- image: "ghcr.io/kubewarden/*"
+  repo: "app-example"
```

Comment:
```suggestion
  repo: "app-example" # optional
```

I would also invert the line order of `repo` and `owner`.

Reply:
I like the idea of the example with a signed pod. I'm not sure about putting GitHub Actions first, not everyone is using GitHub. Maybe url prefix that is more generic? or even public keys that is not experimental

Reply:
I'm happy with https://github.com/kubewarden/verify-image-signatures/pull/25/commits/428803d69ee6aa426501262310167a55f943ae65, resolving!